web/controllers/user_controller.ex

```diff
@@ -34,6 +35,23 @@ defmodule CodeCorps.UserController do
     end
   end
 
+  def github_connect(conn, code) do
+    current_user = Guardian.Plug.current_resource(conn)
+    with {:ok, user} <- Github.connect(current_user, code)
```

Comment:
`Github.connect` ought to return the updated user, which has the `github_auth_token` field, so that's what happens.